### ISSUE TITLE
Graphically show blocked and blocking tasks

### ISSUE THREAD

**Issue by mkauzlar** [kanboard/kanboard#2149]
### Expected behaviour

Please add the possibility to show graphically the links between blocking and blocked tasks in the Board view. If there are several columns with many tasks it would be handy to be able to show all the blocking tasks related to the blocked ones. Perhaps using lines across the Board or something.
### Actual behaviour

At the moment there is an icon in the task on the Board view which tells you that a task has a link to another. You can hover over the icon and it tells you which task is linked to. Then you have to search across the board to find this other task. If you have multiple columns and swimlanes and don't know where the other task is you have to read the whole board.
### Configuration

```
Application version: 1.0.27
PHP version: 5.5.9-1ubuntu4.14
PHP SAPI: fpm-fcgi
OS version: Linux 3.13.0-66-generic
Database driver: sqlite
Database version: 3.8.2
Browser: Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:45.0) Gecko/20100101 Firefox/45.0
```

**Comment by oliviermaridat** [comment]
This is related to #1114 and #1190.
For sure Kanboard could make a great use of task link, but it is not very easy to define how. If you have any ideas...

I made some tests to use the "is blocked by" link to modify "Start date" in the Gantt view. It was very useful, but the inheritance was still missing.

**Comment by mkauzlar** [comment]
I agree with #1114 and this is a basic feature of PM and Gantt charts.
For the Board I haven's seen this feature so far and here is my idea:
Attached is a screenshot of an example Board. In KB-dep2.jpg you can see how, for example, blocking and blocked cards can be shown (red means it's blocking another task, yellow means is blocked by another task). By hovering over over task "#43 Deploy2", the program could draw arrows presenting where the blocking is coming from and where is going to (KB-dep3.jpg). 
![kb-screenshot1](https://cloud.githubusercontent.com/assets/4133383/14714050/ce22f960-07e3-11e6-91e5-425eecd1a342.jpg)

![kb-dep2](https://cloud.githubusercontent.com/assets/4133383/14714051/ce2381be-07e3-11e6-807d-3b1137cce4db.jpg)

![kb-dep3](https://cloud.githubusercontent.com/assets/4133383/14714052/ce260d44-07e3-11e6-93f6-109d816cb080.jpg)

**Comment by xavividal** [comment]
You can use the Relationgraph plugin to see a graph between tasks and which kind of relations exists between them.

**Comment by bugwatcher** [comment]
This issue has been closed automatically for inactivity after more than 1 year. If you still need this feature, you should consider contributing to this open-source project.
